lib/simplecov_helper.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "active_support/inflector"
+require "simplecov"
+
+class SimpleCovHelper
+  DEFAULT_BASE_DIR = "./coverage/"
+
+  def self.report_coverage(base_dir: DEFAULT_BASE_DIR, minimum_coverage: 100)
```

Comment:
Is this going to enforce coverage at 100%, the way DLS does it? Are we ready to commit to that as a team?